service/pom.xml

```diff
@@ -229,11 +229,6 @@
             <artifactId>jgrapht-core</artifactId>
             <version>1.5.2</version>
         </dependency>
-        <dependency>
-            <groupId>com.h2database</groupId>
-            <artifactId>h2</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
Warum wird die H2 Dependency komplett entfernt, wenn wir im Code extra Abfragen darauf machen?

Reply:
Die H2 Abhängigkeit ist in dieser POM bereits in Zeile 63 eingetragen (scope runtime). 
Runtime wird benötigt, damit das CAS auch im "normalen" Betrieb mit H2 Datenbank gestartet werden kann.

https://github.com/minova-afis/aero.minova.cas/pull/846/files/70bfc73ba6248ea40678bd58df8dd3eeed55eb69..a53032ffcbbf29082dc5f4a279754bcba33c489d#diff-e111dd0e82f440d2675d1ad5b10fe3ffde1c976ac42f203065af98550372b13eL63